examples/js/exporters/GLTFExporter.js

```diff
@@ -896,16 +896,16 @@ THREE.GLTFExporter.prototype = {
 
 			}
 
-			if ( ! outputJSON.materials ) {
+			if ( material.isShaderMaterial ) {
```

Comment:
Does `GLTFSpecularGlossinessMaterial` not set `isShaderMaterial` anymore?

Reply:
It's now derived from `MeshStandardMaterial`.